src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -111,6 +151,15 @@ public function handle_submit() {
 		return $result;
 	}
 
+	public function handle_import_finalize( $data ) {
+		$record = Tribe__Events__Aggregator__Records::instance()->get_by_import_id( $post_data['import_id'] );
+
+		do_action( 'debug_robot', '$data :: ' . print_r( $data, TRUE ) );
```

Comment:
These are temporary for my dev logging and will go away when this portion is coded.